debian/control

```diff
@@ -16,14 +16,15 @@ Description: A common package which contains files which are shared by several C
 Package: cloudstack-management
 Architecture: all
 Depends: ${misc:Depends}, ${python:Depends}, openjdk-8-jre-headless | java8-runtime-headless | java8-runtime, cloudstack-common (= ${source:Version}), tomcat6 | tomcat7, sudo, jsvc, python-mysql.connector, libmysql-java, augeas-tools, mysql-client, adduser, bzip2, ipmitool, lsb-release
+Recommends: init-system-helpers (>= 1.14)
```

Comment:
@rhtyd Sorry I just noticed your comment.  I will remove all 3 lines
Recommends: init-system-helpers (>= 1.14)

I am not sure if it will cause any issue. Please test it.